### DISCUSSION TITLE
Plus sign in email addresses breaks the Markdown parser

### DISCUSSION THREAD

**Discussion by schulle4u** [datenstrom/community#355] [Report a bug] [answered]
Hello, 

I'm not sure what causes the following problem, maybe this is a general markdown/parsedown bug and should not reported here...  
I have a mailing list hosted at groups.io where users can subscribe either via web or with an empty email to a specific address, and would like to put an email link on my website. The email address has the pattern `listname+subscribe@groups.io`. When I try to enter this in markdown, `[listname+subscribe@groups.io](mailto:listname+subscribe@groups.io)`, the parser cannot correctly parse the address. As it seems the + sign breaks the email function. I've tried several variants, including writing down the address without markdown linktags, putting the address in quotes and so on. Even inline HTML doesn't work as expected, only if I mask the label, e.g. `<a href="mailto:listname+subscribe@groups.io">listname+subscribe (at) groups.io</a>`. 

Is there a trick to solve this? Tested it with the experimental parsedown plugin as well, same problem there. My [antispam plugin](https://github.com/schulle4u/yellow-plugin-antispam) has a slightly different behaviour, in the email client the + sign is converted into whitespace after clicking the link. 

Steffen

**Comment by markseuffert** [comment, marked as answer]
Hello Steffen, this was a bug in the Markdown plugin. Should be fixed now.

**Comment by schulle4u** [comment]
Thank you, works now.
